# HG changeset patch
# User Rik <rik@octave.org>
# Date 1450132497 28800
#      Mon Dec 14 14:34:57 2015 -0800
# Branch stable
# Node ID a7ebc9815d71485a23c7272d085689ca91493c7f
# Parent  9fe6db1dcc4619b88e8df36518f88adb1108350f
doc: Start help text with " -- ", not " -- : ".

* __makeinfo__.m: Use regexprep to replace " -- : " generated by Texinfo with
" -- ".

diff --git a/scripts/help/__makeinfo__.m b/scripts/help/__makeinfo__.m
--- a/scripts/help/__makeinfo__.m
+++ b/scripts/help/__makeinfo__.m
@@ -143,19 +143,23 @@ function [retval, status] = __makeinfo__
 
     ## Clean up extra newlines generated by makeinfo
     if (strcmpi (output_type, "plain text"))
       if (numel (retval) > 2 && retval(end-1:end) == "\n\n")
         retval = retval(1:end-2);
       endif
     endif
 
+    ## Clean up start of @deftypefn expansion which includes extra ':'
+    retval = regexprep (retval, '^ -- : +', ' -- ', "lineanchors");
+
   unwind_protect_cleanup
     if (exist (name, "file"))
       delete (name);
     endif
   end_unwind_protect
+
 endfunction
 
 
 ## No test needed for internal helper function.
 %!assert (1)
 
